pkg/controller/utils/utils.go

```diff
@@ -184,16 +184,22 @@ func CheckLicenseKey(ctx context.Context, cli client.Client) error {
 	return cli.Get(ctx, DefaultInstanceKey, instance)
 }
 
-// ValidateCertPair checks if the given secret exists and if so
-// that it contains key and cert fields. If a secret exists then it is returned.
+// ValidateCertPair validates the cert pair secret in the tigera-operator namespace.
+func ValidateCertPair(client client.Client, certPairSecretName, keyName, certName string) (*corev1.Secret, error) {
+	return ValidateCertPairInNamespace(client, render.OperatorNamespace(), certPairSecretName, keyName, certName)
+}
+
+// ValidateCertPairInNamespace checks if the given secret exists in the given
+// namespace and if so that it contains key and cert fields. If a secret exists then it is returned.
 // If there is an error accessing the secret (except NotFound) or the cert
 // does not have both a key and cert field then an appropriate error is returned.
 // If no secret exists then nil, nil is returned to represent that no cert is valid.
-func ValidateCertPair(client client.Client, certPairSecretName, keyName, certName string) (*corev1.Secret, error) {
+// TODO: Replace this with version of ValidateCertPair taking in a ns.
```

Comment:
Yeah, I understand why this is appealing, but generally not a huge fan of proliferating different functions for the same thing. We should just do the work up-front to refactor the existing function to take a namespace if that is what we need.

I know it touches a bunch of other places, but it's better than contributing debt (albeit just a tiny bit).

Reply:
@caseydavenport I had proposed doing this so that could minimise the changes made just before the release.

Reply:
Sounds good, although the changes should really be trivial and without much risk. I'm happy to see it done in a follow-up PR so long as it's in the next day or two.